### PR TITLE
CUMULUS-3920 :  Release 18.5.1 backports

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-3920](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3920)

## Changes

Release 18.5.1

Backports for:
 #3810
 #3791
 #3818 
 #3832
 #3807 

This will be released as 18.5.1, with a separate PR for the version up. 
## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: audit-ci.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "http-cache-semantics",
     "lodash.pick",
     "hoek",
+    "jsonpath-plus",
     "semver",
     "axios"
   ]
```
Comment (jennyhliu) [inline audit-ci.json:19]: So my commit removed many entries, and it's not reflected here.
https://github.com/nasa/cumulus/commit/a31481b9ae341b961217976f218aea649e7aa709#diff-3067268e04946afc99fcc6d204f22cc7105a282f106c6eda212a85d8dd3e5ba2L12

File: packages/aws-client/package.json [hunk 2]
```diff
@@ -83,7 +82,7 @@
     "uuid": "^8.2.0"
   },
   "devDependencies": {
-    "@cumulus/test-data": "18.0.0",
+    "@cumulus/test-data": "18.3.5",
```
Comment (jennyhliu) [inline packages/aws-client/package.json:85]: Should this be "18.5.0"?

File: audit-ci.json (hunk 1 above)
Comment (Jkovarik) [reply audit-ci.json:19]: @jennyhliu  good catch, I ... think.   Part of the confusion here is that there were commits in some of the backports (in order) that added them back.   I'm going to remove them and double check audit passes.     jsonpath-plus needs to be in the list though.

Comment (Jkovarik) [reply audit-ci.json:14]: 81eb09b13f3

File: packages/aws-client/package.json (hunk 2 above)
Comment (Jkovarik) [reply packages/aws-client/package.json]: Yes!  😓      146ac9c2a3d

**Comment by Jkovarik** [comment]
@jennyhliu  thanks for the review comments 🙇🏻  - I put this back into review as I believe I addressed both of them.

**Review by jennyhliu — APPROVED** [review]
🌟

**Pull request merged**